src/components/tx/GasParams/index.tsx

```diff
@@ -60,30 +62,28 @@ const GasParams = ({
 
   return (
     <Accordion elevation={0} onChange={onChangeExpand}>
-      <Track {...MODALS_EVENTS.ESTIMATION} label={isAccordionExpanded ? 'Close' : 'Open'}>
-        <AccordionSummary>
-          {isExecution ? (
-            <Typography display="flex" alignItems="center" justifyContent="space-between" width={1}>
-              <span>Estimated fee </span>
-              {isLoading ? (
-                <Skeleton variant="text" sx={{ display: 'inline-block', minWidth: '7em' }} />
-              ) : (
-                <span>
-                  {totalFee} {chain?.nativeCurrency.symbol}
-                </span>
-              )}
-            </Typography>
-          ) : (
-            <Typography>
-              Signing transaction with nonce&nbsp;
-              {nonce || <Skeleton variant="text" sx={{ display: 'inline-block', minWidth: '2em' }} />}
-            </Typography>
-          )}
-        </AccordionSummary>
-      </Track>
+      <AccordionSummary>
+        {isExecution ? (
+          <Typography display="flex" alignItems="center" justifyContent="space-between" width={1}>
+            <span>Estimated fee </span>
+            {isLoading ? (
+              <Skeleton variant="text" sx={{ display: 'inline-block', minWidth: '7em' }} />
+            ) : (
+              <span>
+                {totalFee} {chain?.nativeCurrency.symbol}
+              </span>
+            )}
+          </Typography>
+        ) : (
+          <Typography>
+            Signing transaction with nonce&nbsp;
+            {nonce || <Skeleton variant="text" sx={{ display: 'inline-block', minWidth: '2em' }} />}
+          </Typography>
+        )}
+      </AccordionSummary>
 
       <AccordionDetails>
-        <GasDetail isLoading={nonce === undefined} name="Nonce" value={(nonce ?? '').toString()} />
+        {nonce && <GasDetail isLoading={false} name="Nonce" value={nonce.toString()} />}
```

Comment:
`{nonce !== undefined &&`

Reply:
Good catch! Could go wrong if we have a nonce of 0 🙈